canonical/identity-platform-admin-ui
# WD-13638 - feat: log in via OIDC

## Changes

- Set up component testing.
- Check if authenticated via the API.
- Login using OIDC.

https://warthogs.atlassian.net/browse/WD-13638

## QA

- Set up the UI and identity backend, following these instructions: https://github.com/canonical/identity-platform-admin-ui/wiki/Dev-UI-with-OIDC.
- Visit http://localhost:8000/api/v0/auth/logout to make sure you're logged out.
- Visit http://localhost:8000/ui
- You should see the login screen.
- Click the login button and after logging you in you should get returned to the UI which should show you as logged in.
